src/slowlog.c

```diff
@@ -53,7 +53,7 @@ slowlogEntry *slowlogCreateEntry(robj **argv, int argc, long long duration) {
     se->argc = slargc;
     se->argv = zmalloc(sizeof(robj*)*slargc);
     for (j = 0; j < slargc; j++) {
-        /* Logging too many arguments is a useless memory waste, so we stop
+        /* Logging too many arguments is an useless memory waste, so we stop
```

Comment:
```suggestion
        /* Logging too many arguments is a useless memory waste, so we stop
```
are you sure?
maybe by English is not a good as yours...